integration/fuse/src/test/java/alluxio/jnifuse/struct/FuseFileInfoTest.java

```diff
@@ -26,8 +27,9 @@
 public class FuseFileInfoTest {
   @Test
   public void offset() {
+    Configuration.set(PropertyKey.FUSE_JNIFUSE_LIBFUSE_VERSION, 2);
     LibFuse.loadLibrary(AlluxioFuseUtils.getLibfuseVersion(Configuration.global()));
-    FuseFileInfo jnifi = FuseFileInfo.of(ByteBuffer.allocate(256));
+    FuseFileInfo jnifi = FuseFileInfo.of(ByteBuffer.allocate(40));
```

Comment:
Why are we changing from 256 to 40 here?

Reply:
because Fuse File Info after my debug log is 40 bytes in both fuse 2 and fuse 3. It's constant size.
Previously we do not know it's exact size so set it to a bigger value. (A lazy settings)